pulp_smash/tests/pulp2/rpm/api_v2/test_iso_crud.py

```diff
@@ -362,3 +364,70 @@ def test_invalid_file_feed(self):
                 self.assertEqual(info['error']['response_code'], 404)
                 self.assertEqual(info['error']['response_msg'], 'Not Found')
                 self.assertIn(info['name'], missing)
+
+
+class ISOUpdateTestCase(unittest.TestCase):
+    """Verify how ISO repos handles changes in content already in Pulp."""
+
+    def test_all(self):
+        """Verify how ISO repos handles changes in content already in Pulp.
+
+        The main goal of this test is to verify how ISO repository handles
+        updates in content already in Pulp.
+
+        For this test two different feed urls will be used.
+        These urls contain the same amount of units, the units have the same
+        type and name in both urls, but different content thereafter different
+        checksum values.
+        To recreate this dynamic scenario of change. After the repository is
+        synced for the first time the feed url is updated, and the repository
+        is synced again.
+
+        This test targets the following issues:
+
+        * `Pulp Smash #715 <https://github.com/PulpQE/pulp-smash/issues/715>`_
+        * `Pulp #2773 <https://pulp.plan.io/issues/2773>`_
+        * `Pulp #3047 <https://pulp.plan.io/issues/3047>`_
+        * `Pulp #3100 <https://pulp.plan.io/issues/3100>`_
+
+        Do the following:
+
+        1. Create and sync an ISO repository.
+        2. To recreate the scenario of change in units already in Pulp. After
+           the repository is synced, the feed url will be updated, and the
+           repository will be synced again.
```

Comment:
The reader already knows that this is happening after the repository has been synced, because step one says that the repository has been synced. Something like this would read more nicely:

> 2. Update the repository's feed URL, and sync it. This simulates a change in the contents of the source ISOs.